plugins/sign-on/openid-connect-bcc-customizations/bcc-signon.php

```diff
@@ -191,7 +211,25 @@ public static function get_access_token(){
 
 		return $tokens['access_token'];
 	}
+
+	/**
+     * Delete all subscribers
+     */
+	public static function delete_subscribers() {
```

Comment:
You could check if current user is an admin before deleting subscribers

Reply:
I'm not sure that's necessary. See my comment above.